.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/*.yml'
+
+jobs:
+  actionlint:
+    name: Lint Actions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Actionlint
+        env:
+          ACTIONLINT_VERSION: 1.6.12
+        run: |
+          wget -q -O- "https://github.com/rhysd/actionlint/releases/download/v${ACTIONLINT_VERSION}/actionlint_${ACTIONLINT_VERSION}_linux_amd64.tar.gz" | tar -x -z -C . actionlint && \
```

Comment:
why does this download and execute a binary?

Reply:
Because thats the only way to execute the linter

Reply:
Are there other ways to use this that do not involve executing arbitrary downloaded binaries in a CI env? The possible risk outweighs the benefits imho

Reply:
Thats mostly how CI jobs seem to work as far as I can tell. The action downloads / installs dependencies required for the work to be done, then executes it. Is it for instance very different to installing `prettier`?

Reply:
There are other ways of achieving the same, but they are all downloading the binary: https://github.com/rhysd/actionlint/blob/main/docs/usage.md#tools-integration

Reply:
yes, prettier is loaded from npm which does not allow republishing packages with the same version and any funky prettier release would be found out almost immediately due to its popularity.
The github release can be overwritten so it may work fine today but not tomorrow.

Funnily enough prettier itself seems to use  node-actionlint https://github.com/prettier/prettier/blob/66585d9f250f11569456421e66a2407397e98f69/package.json#L155

Reply:
Do you want me to switch to that?

Reply:
thats even less used than actionlint itself so i'd be still hesitant to add it to a CI with access to secrets. Running it in a properly secured local env and PRing any findings seems like a safer way to use this.

Reply:
I'll close the PR then.